fabric-convention-tags-v2/src/main/java/net/fabricmc/fabric/api/tag/convention/v2/ConventionalEnchantmentTags.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.api.tag.convention.v2;
+
+import net.minecraft.enchantment.Enchantment;
+import net.minecraft.registry.tag.TagKey;
+
+import net.fabricmc.fabric.impl.tag.convention.v2.TagRegistration;
+
+public final class ConventionalEnchantmentTags {
+	private ConventionalEnchantmentTags() {
+	}
+
+	/**
+	 * A tag containing enchantments that increase the amount or
+	 * quality of drops from blocks, such as {@link net.minecraft.enchantment.Enchantments#FORTUNE}.
+	 */
+	public static final TagKey<Enchantment> INCREASES_BLOCK_DROPS = register("fortune");
```

Comment:
Since this is somewhat a breaking change, do we want to change the id `fortune` -> `increaces_block_drops`? Same for entity.

Reply:
Agreed! Matches the other enchantment tags better and is more clear that it is to be used for more than just fortune/looting enchantments. Made the change now and added the old tag to the warnings